doc/protocol.md

```diff
@@ -153,63 +151,63 @@ Exclusion criteria:
 
 ### Stopping criteria
 
-In terms of database searches, we run the final search 01/10/22. Hand searching
+In terms of database searches, we aim to run the final search by 01/10/22. Searches
 will be concluded when we reach conceptual saturation (i.e., when it is
 determined that we are not identifying any new concepts/resources). The first
-1000 records will be screened on Google.
+1000 records will be screened on Google. In the interests of feasibility, a maximum for 4 books will be selected and evaluated. Books will be selected based on a consensus in the research group.
 
 DANIEL: why not search all the references? Do we want to have two people
 review each paper or is it okay we search a database each? For this
 study, I would think that it is okay not to have two reviewers, as it is
 not a systematic reivew. It is more important to have more databases,
 then double check each reference.
+Hannah: I think I added this "1000 records" idea based on my discussions with the research librarian, who had done something similar. I'm happy to search all references. I agree with your proposal to search a database each, although maybe we should trial this together to ensure that we are using a similar approach/decision-making method.
 
 ## Analysis Plan
 
 ### Data analysis approach
 
+Data from all searches in online databases will be exported into text files or
+CVS files for further processing including removal of duplicates. Google Sheets
+will be used to manage records including screening decisions.
```

Comment:
Not sure yet if Google Sheets will be the best approach, but it might work if we can export the results back into the Git repository.